src/OrchardCore.Modules/OrchardCore.Lists/Controllers/OrderController.cs

```diff
@@ -53,7 +53,7 @@ public async Task<IActionResult> UpdateContentItemOrders(string containerId, int
             }
 
             // Include draft items.
-            var pageOfContentItems = (await _containerService.QueryContainedItemsAsync(containerId, true, pager, ContentsStatus.None, null)).ToList();
+            var pageOfContentItems = (await _containerService.QueryContainedItemsAsync(containerId, true, pager, null)).ToList();
```

Comment:
Itt meg a `ListPartDisplayDriver`-be nis `null` van átadva az utolsó paraméternek. Pedig itt korábban `false` volt, míg ott `true`, szóval az egyik hibás.

Kérlek nézd végig a PR-odat, sorról rorra, hogy mi változott, és csak olyat változtattál, amit tényleg akartál.

Reply:
Nekem sajnos csak az jut eszembe hogy visszaírni a ContentsStatus-t az interface-be, mert a ContainedItemOptions-el nem tudom hogy tudnék az eredetileg bool változóval egyező funkciójú paramétert átadni.

Reply:
Benne a statust használhatod. A `CreateDefaultContentIndexFilter()`-t ugye már átírtad a `bool publishedOnly`-ról, szóval ezt a változtatást kellene itt is lekövetni.

Nézd meg hogy mit változtattál a PR alatt, hogy össze tudd vetni a régi logikát az újjal: https://github.com/Lombiq/OrchardCore/pull/3/files

Reply:
Itt azt a logikát használtam, hogy ami false volt az status.None, ami true az status.Published. Tehát orderController-ben, és DisplayDriver-ben is beírok egy plusz paramétert ami vagy status.None vagy status.Published, ahogy tegnap volt?
Igazából engem az zavart össze, hogy IContainerService interface-ből ki kellett törölni a ContentsStatus típusú paramétert.

Reply:
Nem kell plusz paraméter, hiszen ez az információ benne van a `ContainedItemOptions`-ben, azt kellene használni.

Reply:
Azt találtam ki, hogy DisplayDriver-be visszaírom a containedItemOptions példányát, annak Status property-jét Published-re állítom, így ha containedItemOptions-t adok át metódusparaméternek, akkor ott van benne a true-val egyenértékű published.
Az orderController-ben pedig létrehozok egy ugyanilyen példányt és simán átadom. A status property alapértelmezetten None, ami megfelel a false-nak.

Reply:
Jól hangzik.